examples/reference-entity-model/entities/Order.entity.json

```diff
@@ -1,31 +1,46 @@
-{
-  "info": {
-    "title": "Order",
-    "version": "0.0.1",
-    "baseUri": "http://marklogic.com/example/"
-  },
-  "definitions": {
-    "Order": {
-      "required": [ ],
-      "primaryKey": "orderId",
-      "properties": {
-        "orderId": {
-          "datatype": "string"
-        },
-        "address": {
-          "$ref": "#/definitions/Address"
-        }
-      }
-    },
-    "Address": {
-      "properties": {
-        "city": {
-          "datatype": "string"
-        },
-        "state": {
-          "datatype": "string"
-        }
-      }
-    }
-  }
-}
+{
+  "info": {
+    "title": "Order",
+    "version": "0.0.1",
+    "baseUri": "http://marklogic.com/example/"
+  },
+  "definitions": {
+    "Order": {
+      "primaryKey": "orderId",
+      "properties": {
+        "orderId": {
+          "datatype": "integer"
+        },
+        "orderTime": {
+          "datatype": "dateTime"
+        },
+        "orderedBy": {
+          "datatype": "integer",
+          "relatedEntityType": "http://example.org/Customer-0.0.1/Customer",
+          "joinPropertyName": "customerId"
+        },
+        "deliveredTo": {
+          "datatype": "integer"
+        },
+        "lineItems": {
+          "datatype": "array",
+          "items": {
+            "$ref": "#/definitions/LineItem"
+          }
+        }
+      }
+    },
+    "LineItem": {
+      "properties": {
+        "quantity": {
+          "datatype": "integer"
+        },
+        "orderIncludes": {
```

Comment:
Same thing, let's do "includesProduct". Actually, let's do "hasProduct" - "has" is more typical here.